src/mesido/workflows/emerge.py

```diff
@@ -0,0 +1,246 @@
+        options["solver"] = "highs"
+        return options
+
+    def times(self, variable=None):
+        return super().times(variable)[:25]
+
+    def energy_system_options(self):
+        """
+        This function does not add anything at the moment but during debugging we use this.
+
+        Returns
+        -------
+        Options dict for the physics modelling
+        """
+        options = super().energy_system_options()
+        options["minimum_velocity"] = 0.0
+        options["heat_loss_disconnected_pipe"] = False
+        options["neglect_pipe_heat_losses"] = False
+        options["include_asset_is_switched_on"] = True
+        options["include_electric_cable_power_loss"] = False
+        return options
+
+    def post(self):
+        # In case the solver fails, we do not get in priority_completed(). We
+        # append this last priority's statistics here in post().
+        # TODO: check if we still need this small part of code below
+        success, _ = self.solver_success(self.solver_stats, False)
+        # if not success:
+        #     time_taken = time.time() - self.__priority_timer
+        #     self._priorities_output.append(
+        #         (
+        #             self.__priority,
+        #             time_taken,
+        #             False,
+        #             self.objective_value,
+        #             self.solver_stats,
+        #         )
+        #     )
+
+        super().post()
+
+        # Optimized ESDL
+        # self._write_updated_esdl(self._ESDLMixin__energy_system_handler.energy_system)
+        #
+        # self._save_json = False
+        #
+        # if os.path.exists(self.output_folder) and self._save_json:
+        #     self._write_json_output()
+        #
+        # results = self.extract_results()
+        #
+        # for _type, assets in self.energy_system_components.items():
+        #     for asset in assets:
+        #         print("----------------------------------")
+        #         print(f"{asset} financials:")
+        #         try:
+        #             print(f"revenue of {asset} in MEUR/day: ", results[f"{asset}__revenue"] / 1e6)
+        #         except KeyError:
+        #             print(f"{asset} does not have a revenue")
+        #             pass
+        #         try:
+        #             print(
+        #                 f"fixed operational costs of {asset} in MEUR/yr : ",
+        #                 results[f"{asset}__fixed_operational_cost"] / 1e6,
+        #             )
+        #             print(
+        #                 f"variable operational costs of {asset} : ",
+        #                 results[f"{asset}__variable_operational_cost"] / 1e6,
+        #             )  # not yet all included in financialmixin
+        #             print(f"max size of {asset} : ", results[f"{asset}__max_size"])
+        #         except KeyError:
+        #             print(f"{asset} does not have a costs")
+        #             pass
+
```

Comment:
uncomment this part where the results are printed/saved

Reply:
agree done